kmir/src/kmir/utils.py

```diff
@@ -16,11 +16,44 @@
 from pyk.proof import APRBMCProof, APRBMCProver, APRProof, APRProver
 from pyk.proof.equality import EqualityProof, EqualityProver
 from pyk.proof.proof import Proof, ProofStatus
-from pyk.utils import BugReport
+from pyk.utils import BugReport, single
+
+T1 = TypeVar('T1')
+T2 = TypeVar('T2')
```

Comment:
Can these use the dedicated syntax for type variables? https://docs.python.org/3/library/typing.html#typing.TypeVar

Reply:
Our project is using python 3.10, and when I tried to implement the change according to the documentation I ran into this error `Function type parameter syntax requires Python 3.12 or newer Pylance`.